packages/dialect-react-ui/components/Icon/ArrowClockwise.tsx

```diff
@@ -0,0 +1,18 @@
+import type { SVGProps } from 'react';
+
+const SvgComponent = (props: SVGProps<SVGSVGElement>) => (
+  <svg
+    width={12}
+    height={13}
+    fill="none"
```

Comment:
Please add `viewBox` and `preserveAspectRatio="xMidYMid meet"` props to our SVGs in case we need to scale them.